pom.xml

```diff
@@ -429,6 +415,14 @@
                     <artifactId>maven-release-plugin</artifactId>
                     <version>3.0.0-M5</version>
                 </plugin>
+                <plugin>
+                     <groupId>org.codehaus.mojo</groupId>
+                     <artifactId>versions-maven-plugin</artifactId>
+                     <version>2.10.0</version>
+                     <configuration>
+                         <rulesUri>file:\\\\${project.basedir}/versionRuleset.xml</rulesUri>
+                     </configuration>
+                </plugin>
```

Comment:
Note that this uses the versionRuleset.xml to exclude updates when performing update checks with the maven versions plugin.

Enables visibility of updates through maven with the following commands:
```
#!/bin/bash
mvn versions:display-dependency-updates -U
mvn versions:display-plugin-updates -U
```

Reply:
I've been running that shell script on the weekends to keep everything up-to-date.  I think it's been working pretty well overall.